src/main/kotlin/com/cognifide/gradle/aem/common/instance/local/QuickstartResolver.kt

```diff
@@ -20,6 +20,13 @@ class QuickstartResolver(private val aem: AemExtension) {
      */
     val jarUrl = aem.obj.string {
         aem.prop.string("localInstance.quickstart.jarUrl")?.let { set(it) }
+        aem.prop.string("localInstance.quickstart.dirUrl")?.let {
+            set(
+                File(it).walk()
```

Comment:
if walk is recursive this is potentially dangerous, maybe listFiles() will be safer